node/src/chain_spec.rs

```diff
@@ -73,6 +77,15 @@ pub fn development_config() -> Result<ChainSpec, String> {
 				"0xec00ad0ec6eeb271a9689888f644d9262016a26a25314ff4ff5d756404c44112"
 					.parse()
 					.unwrap(),
+				// Ethereum AppId for SnowBridged Assets
+				vec![
+						(
+							App::ETH, H160::from_slice(&hex!["Fc97A6197dc90bef6bbEFD672742Ed75E9768553"][..]).into()
```

Comment:
Are these addresses for local eth testnet of for kovan testnet?

Reply:
Thanks for catching that. Changed it to Kovan testnet values.